src/components/symptom-shark/presentational/NotesTimeline/NotesTimeline.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext } from 'react';
+import "./NotesTimeline.css"
+import { format, startOfMonth } from 'date-fns';
+import { DateRangeContext, TextBlock, getDayKey, language } from '../../../..';
+import { SymptomSharkVisualizationContext } from '../../container/VisualizationCoordinator/VisualizationCoordinator';
+import { getDatesForMonth } from '../../../../helpers/date-helpers';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircle } from '@fortawesome/free-solid-svg-icons';
+import LogEntryIcon from '../LogEntryIcon/LogEntryIcon';
+
+export interface SymptomSharkNotesTimelineProps {
+    intervalStart?: Date;
+    innerRef?: React.Ref<HTMLDivElement>;
+}
+
+export default function (props: SymptomSharkNotesTimelineProps) {
+    let visualizationContext = useContext(SymptomSharkVisualizationContext);
+    if (!visualizationContext) {
+        return <TextBlock innerRef={props.innerRef}>Error: Notes timeline must be used inside a Symptom Shark Visualization Coordinator.</TextBlock>
+    }
+
+    let dateRangeContext = useContext(DateRangeContext);
+    let intervalStart = props.intervalStart || startOfMonth(new Date());
+    if (dateRangeContext) {
+        intervalStart = dateRangeContext.intervalStart;
+    }
+
+    if (!visualizationContext.symptoms.length && !visualizationContext.treatments.length) {
+        return null;
+    }
+
+    var monthDays = getDatesForMonth(intervalStart.getFullYear(), intervalStart.getMonth());
+    var daysWithNotes = monthDays.filter((m) => visualizationContext!.logEntries[getDayKey(m)]?.notes?.length);
```

Comment:
Minor:  Unnecessary parens around `m`.  Also, `m` could probably be renamed to something more descriptive.